service/mapper/transitions_test.go

```diff
@@ -0,0 +1,1078 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package mapper
+
+import (
+	"io"
+	"sync"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/ledger/complete/mtrie/trie"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/optakt/flow-dps/models/dps"
+	"github.com/optakt/flow-dps/testing/mocks"
+)
+
+// For path/payload/trie/trie update test variables, see testing globals defined in pathings_test.go.
+var (
+	testHeight = uint64(42)
```

Comment:
I disagree with that style personally, because I think the types should either be all in front of the `=`, or all behind it. As with structs, they have to be behind, I prefer to always use the notation used by @Ullaakut here, though I think he also preferred the other one in the past ;)

Reply:
For my brain this reads - this is a variable `testHeight`, which is an `uint64`, and it's value is 42.
The other thing is - I'm casting 42 to `uint64`, and the variable `testHeight` will infer the type from there.

The first reads like language, the second as a _programming_ language.

But I do like uniformity, so the struct argument is making me second guess now, even if it still looks nicer :/

Reply:
For me personally, I don't mind tbh 😄

Reply:
There was a blog post written about this once. From what I remember, using `var name type` should be used to indicate we explicitly want to use the zero value of a type, and `name := type(value)` should be used when we want to initialize the value. That makes the distinction the clearest and should thus be preferred.